CHANGELOG.md

```diff
@@ -12,6 +12,11 @@
 ### Breaking Changes
 ...
 
+## 7.2.0
+
+### Fixes
+- Update names of `pharia-1` models to lowercase, aligning with fresh deployments of the api-scheduler.
```

Comment:
Should this be under the `## Unreleased` heading?

Reply:
Yes. We put everything under `## Unreleased` until we create the actual release. Please move it up and remove the `## 7.2.0 section`